packages/plugin/src/api/analytics.js

```diff
@@ -106,6 +113,39 @@ export const apiFetchAnalyticsForYearOverYear = async (
     }))
 }
 
+const prepareRequestsForRelativeDays = async ({
```

Comment:
Is this an async function? At least I didn't see anything async about it.

Reply:
Good catch. Copy-paste mistake.